mongodbatlas/fw_resource_mongodbatlas_database_user_migration_test.go

```diff
@@ -309,14 +316,15 @@ func TestAccConfigRSDatabaseUser_Migration_WithRoles(t *testing.T) {
 	})
 }
 
-func TestAccConfigRSDatabaseUser_Migration_WithScopes(t *testing.T) {
+func TestAccMigrationConfigRSDatabaseUser_WithScopes(t *testing.T) {
 	var (
-		resourceName = "mongodbatlas_database_user.test"
-		username     = acctest.RandomWithPrefix("test-acc-user-")
-		password     = acctest.RandomWithPrefix("test-acc-pass-")
-		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName  = acctest.RandomWithPrefix("test-acc")
-		clusterName  = acctest.RandomWithPrefix("test-acc-cluster")
+		resourceName          = "mongodbatlas_database_user.test"
+		username              = acctest.RandomWithPrefix("test-acc-user-")
+		password              = acctest.RandomWithPrefix("test-acc-pass-")
+		orgID                 = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName           = acctest.RandomWithPrefix("test-acc")
+		clusterName           = acctest.RandomWithPrefix("test-acc-cluster")
+		lastVersionConstraint = os.Getenv("MONGODB_ATLAS_LAST_VERSION")
```

Comment:
do we need to check that this env var exists?

Reply:
nice catch, will update

Reply:
updated